package.json

```diff
@@ -47,15 +47,16 @@
   },
   "dependencies": {
     "@fortawesome/fontawesome-svg-core": "^1.2.36",
+    "@fortawesome/free-brands-svg-icons": "^6.1.1",
```

Comment:
Is this used? If not let's remove it

Reply:
sure.  I assume I can just delete this line from package.json?  I did that.